src/open_order.py

```diff
@@ -1,11 +1,55 @@
 from __future__ import annotations
 
-from typing import List
+import base64
+from typing import Any, Dict, List
 
 from solana.publickey import PublicKey
 from solana.rpc.api import Client
+from solana.system_program import CreateAccountParams, create_account
 
 from ._layouts.open_orders import OPEN_ORDERS_LAYOUT
+from .instructions import DEFAULT_DEX_PROGRAM_ID
+from .utils import load_bytes_data
+
+
+def make_create_account_transaction(
+    owner_address: PublicKey,
+    new_account_address: PublicKey,
+    lamports: int,
+    program_id: PublicKey = DEFAULT_DEX_PROGRAM_ID,
+):
+    return create_account(
+        CreateAccountParams(
+            from_pubkey=owner_address,
+            new_account_pubkey=new_account_address,
+            lamports=lamports,
+            space=OPEN_ORDERS_LAYOUT.sizeof(),
+            program_id=program_id,
+        )
+    )
+
+
+def get_filtered_program_accounts(
+    address: str, program_id: str, filters: Dict[str, Dict[str, Any]]
+) -> List[Dict[str, Any]]:
```

Comment:
Can we not use `Any` here and define a named tuple?

Reply:
Don't think this is possible since they are different types.

Reply:
You can use `Union`